ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml

```diff
@@ -512,7 +512,18 @@ objects:
           update_image_registry &
         fi
 
+        # we need jq for later rhcos url extraction 
+        mkdir /tmp/bin
+        (curl -L -o /tmp/bin/jq https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 2>/dev/null && chmod +x /tmp/bin/jq)
```

Comment:
You're sure that jq isn't in the test image already?

Reply:
yes , i'm positive:
# docker run --entrypoint /bin/bash registry.svc.ci.openshift.org/ci-op-fqvy6k9t/stable:ovirt-installer -c 'find /  | grep jq'   
find: '/proc/tty/driver': Permission denied
find: '/run/cryptsetup': Permission denied
find: '/root': Permission denied
find: '/var/lib/yum/history/2020-07-13/1': Permission denied
find: '/var/lib/yum/history/2020-07-13/2': Permission denied
find: '/var/lib/yum/history/2020-07-13/3': Permission denied
find: '/var/lib/machines': Permission denied
find: '/var/lib/rhsm': Permission denied

Reply:
i agree that downloading it's fragile, using a different approach, I'll copy it from the `lease` container to shared volume.